build_sym.py

```diff
@@ -0,0 +1,78 @@
+#!/usr/bin/env python
+
+"""Compiles a SymbolTable from TSV file."""
+
+import argparse
+import unicodedata
+
+from typing import List
+
+import pynini
+
+
+def _char_processor(token: str) -> List[str]:
+
+    """Returns a list of characters with unseparated diacritics
+    """
+    MODS = frozenset(
+        {
+            "ˡ",
+            "ˍ",
+            "ʲ",
+            "ˠ",
+            "˺",
+            "ː",
+            "˞",
+            "˽",
+            "ˬ",
+            "˖",
+            "ʰ",
+            "ˤ",
+            "˳",
+            "˟",
+            "ⁿ",
+            "ʷ",
+            "˕",
+            "ˌ",
+            "˷",
+            "˔",
+        }
+    )
+    chars = []
```

Comment:
I went with returning lists because the diacritic always comes after the phone, and it seemed easier to combine them with a previous character using `.pop()`, rather than checking for them in the next character and yielding. How do you recommend reworking it to do the latter for the generator?

Reply:
Oh good point. I could do it but it's going to be a bit tricky (you have to yield the previous character if it's NOT followed by a diacritic, or the combination of the two if it is...). So leave it as is.